webui/react/package.json

```diff
@@ -11,9 +11,8 @@
     "postshrinkwrap": "npx npm-force-resolutions && npm install --ignore-scripts",
     "start": "craco start",
     "build": "craco build",
-    "lint": "npm run lint:js && npm run lint:css && npm run format-check",
```

Comment:
I think its nice to have `lint` command to run all cuz there's a [ticket](https://determinedai.atlassian.net/browse/DET-8093) to introduce lint-staged.
and we don't need to run multiple npm commands on local

Reply:
I'd suggest we lean into using make instead of package.json it's more flexible and not unique to the webui module.
`make check` should do what `lint` did with more options

Reply:
lint also was confusing here to me as we had both that and the new set of `format* scripts? defined.

Reply:
I thought adding `lint` is good cuz I use `npm` instead of `make` (its just because I just used to use `npm`)
seems like `lint-staged`can use commands outside of `package.json` so I think we don't need to change it for now.

Reply:
fair point. I added it back